testim-cli.js

```diff
@@ -28,12 +28,7 @@ async function runCommand(params) {
   const gridArg = `--grid="${testimGrid}"`;
   const preparedCommand = trimCommand(`${testimCommand} ${tokenArg} ${projectArg} ${gridArg}`);
 
-  let commandToExecute = "";
-  if (installTestim) {
-    commandToExecute = `{ npm install -g ${TESTIM_NPM_PACKAGE} > /dev/null; } && ${TESTIM_CLI_NAME} ${preparedCommand}`;
-  } else {
-    commandToExecute = `${TESTIM_CLI_NAME} ${preparedCommand}`;
-  }
+  const commandToExecute = `node ${PLUGIN_ABSOLUTE_PATH}/node_modules/${TESTIM_MODULE_PATH} ${preparedCommand}`;
```

Comment:
I remember Matan using the binary at `./node_modules/.bin/testim`. Is there any difference between running this script and running the binary?

Reply:
These are links to the same files